client/assets/js/main.js

```diff
@@ -13,50 +13,60 @@ function handleClickEvents(){
     },3100)
   });
   $('.experience').on('click', function(){ //go to map and camera
-    debugger;
     if(window.outerWidth < 750 ){
-      window.open('map.html')
+      window.open('maptest.html','_self')
```

Comment:
Are we using maptest? or should this be meter?

Reply:
it was maptest the day before

Reply:
No worries. i dont know this file as much. I was just checking in case it was a mistake.